kotlin/workflow-runtime/src/main/java/com/squareup/workflow/WorkflowHost.kt

```diff
@@ -207,9 +208,7 @@ suspend fun <InputT, StateT, OutputT : Any, RenderingT> runWorkflowTree(
   } catch (e: Throwable) {
     // For some reason the exception gets masked if we don't explicitly pass it to cancel the
```

Comment:
Is this still true after the upgrade?

Reply:
Seems like no – the tests all pass if I remove this catch. Sweet!